tests/upgrade/postgres_run.sh

```diff
@@ -165,6 +166,19 @@ test_upgrade_paths() {
 
     touch "${UPGRADE_PROGRESS_POSTGRES_CENTRAL_DB_BOUNCE}"
 
+    ########################################################################################
+    # Upgrade back to latest to run the smoke tests by first walking previous releases     #
+    ########################################################################################
+    for str in ${PREVIOUS_RELEASES[@]}; do
+      info "Walking upgrades -- release => $str"
+      kubectl -n stackrox set image deploy/central "*=$REGISTRY/main:$str"
+      kubectl -n stackrox set image deploy/central-db "*=$REGISTRY/central-db:$str"
+
+      wait_for_api
+      wait_for_central_db
+      info "4.5.0 should fail"
+    done
```

Comment:
How about adding some time for each release walked? It could help us catching more issues.